helpers/request.ts

```diff
@@ -159,10 +159,10 @@ function checkApplicationVersion(response: Response): boolean {
 /**
  * Check that response status is ok
  */
-export function checkStatus(response: Response): Promise<Response> {
+export function checkStatus(response: Response, bypassRedirect = false): Promise<Response> {
```

Comment:
For consistency with other function ?

```suggestion
export function checkStatus(response: Response, bypassRedirect?: boolean): Promise<Response> {
```

Reply:
Yeah I wanted to make it really clear that the default value is false, but I could not do the same for the others since they have an optional parameter before this one.

Reply:
I'll keep it consistent everywhere.